opensrp-chw/build.gradle

```diff
@@ -234,7 +234,7 @@ android {
             versionName "1.0.8"
             buildConfigField "int", "OPENMRS_UNIQUE_ID_INITIAL_BATCH_SIZE", '10000'
             buildConfigField "int", "OPENMRS_UNIQUE_ID_BATCH_SIZE", '10000'
-            buildConfigField "int", "OPENMRS_UNIQUE_ID_SOURCE", '2'
+            buildConfigField "int", "OPENMRS_UNIQUE_ID_SOURCE", '1'
```

Comment:
Why is this changed?

Reply:
it was not pulling the unique ids from source 1

Reply:
I believe this section is under `ba` and not DRC

Reply:
no this is under DRC

Reply:
yes its under Ba it was originally '1' in BA (master branch) i've updated the correction